### PR TITLE
Update rainbow

### PR DESCRIPTION
**Pull request by TomerHawk**

### CODE REVIEW

File: signatures/rainbow/rainbow-standard/impl/avx2.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name: AVX2
+platform: C
+hardware features:
+  - ''
```
Comment (benj-zen) [inline signatures/rainbow/rainbow-standard/impl/avx2.yaml:4]: empty item

File: signatures/rainbow/rainbow-standard/bench/avx2_V_haswell.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+timings:
+  gen: 217000000
+  enc|sign: 857000
+  dec|vrfy: 423000
+  unit: cycles
+memory requirements:
+  gen: 18250000
+  enc|sign: 14200000
+  dec|vrfy: 14700000
+  comment: ''
```
Comment (benj-zen) [inline signatures/rainbow/rainbow-standard/bench/avx2_V_haswell.yaml:13]: empty comment

File: signatures/rainbow/rainbow-standard/bench/avx2_I_haswell.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+timings:
+  gen: 12000000
+  enc|sign: 85000
+  dec|vrfy: 42000
+  unit: cycles
+memory requirements:
+  gen: 12990000
+  enc|sign: 12840000
+  dec|vrfy: 12990000
+  comment: ''
```
Comment (benj-zen) [inline signatures/rainbow/rainbow-standard/bench/avx2_I_haswell.yaml:13]: empty comment

File: signatures/rainbow/rainbow-standard/bench/avx2_III_haswell.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+timings:
+  gen: 60000000
+  enc|sign: 348000
+  dec|vrfy: 162000
+  unit: cycles
+memory requirements:
+  gen: 15300000
+  enc|sign: 13450000
+  dec|vrfy: 13700000
+  comment: ''
```
Comment (benj-zen) [inline signatures/rainbow/rainbow-standard/bench/avx2_III_haswell.yaml:13]: empty comment

File: signatures/rainbow/rainbow-cz/impl/avx2.yaml [hunk 5]
```diff
@@ -0,0 +1,4 @@
+name: AVX2
+platform: C
+hardware features:
+  - ''
```
Comment (benj-zen) [inline signatures/rainbow/rainbow-cz/impl/avx2.yaml:4]: empty item

File: signatures/rainbow/rainbow-cz/bench/avx2_I_haswell.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+timings:
+  gen: 12000000
+  enc|sign: 85000
+  dec|vrfy: 3500000
+  unit: cycles
+memory requirements:
+  gen: 12830000
+  enc|sign: 12840000
+  dec|vrfy: 12830000
+  comment: ''
```
Comment (benj-zen) [inline signatures/rainbow/rainbow-cz/bench/avx2_I_haswell.yaml:13]: empty comment

File: signatures/rainbow/rainbow-cz/bench/avx2_III_haswell.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+timings:
+  gen: 56000000
+  enc|sign: 384000
+  dec|vrfy: 20000000
+  unit: cycles
+memory requirements:
+  gen: 14460000
+  enc|sign: 13450000
+  dec|vrfy: 13110000
+  comment: ''
```
Comment (benj-zen) [inline signatures/rainbow/rainbow-cz/bench/avx2_III_haswell.yaml:13]: empty comment

File: signatures/rainbow/rainbow-compressed/impl/avx2.yaml [hunk 8]
```diff
@@ -0,0 +1,4 @@
+name: AVX2
+platform: C
+hardware features:
+  - ''
```
Comment (benj-zen) [inline signatures/rainbow/rainbow-compressed/impl/avx2.yaml:4]: empty item

File: signatures/rainbow/rainbow-compressed/bench/avx2_I_haswell.yaml [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+timings:
+  gen: 12000000
+  enc|sign: 7600000
+  dec|vrfy: 3500000
+  unit: cycles
+memory requirements:
+  gen: 12830000
+  enc|sign: 12840000
+  dec|vrfy: 12830000
+  comment: ''
```
Comment (benj-zen) [inline signatures/rainbow/rainbow-compressed/bench/avx2_I_haswell.yaml:13]: empty comment

File: signatures/rainbow/rainbow-compressed/bench/avx2_III_haswell.yaml [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+timings:
+  gen: 56000000
+  enc|sign: 38000000
+  dec|vrfy: 20000000
+  unit: cycles
+memory requirements:
+  gen: 14460000
+  enc|sign: 13620000
+  dec|vrfy: 13110000
+  comment: ''
```
Comment (benj-zen) [inline signatures/rainbow/rainbow-compressed/bench/avx2_III_haswell.yaml:13]: empty comment

**Review by benj-zen — CHANGES_REQUESTED** [review]
I think you mixed up the key sizes and some of the benchmarks for the parameter sets. The table captions are below the tables in the paper.

**Comment by TomerHawk** [comment]
> I think you mixed up the key sizes and some of the benchmarks for the parameter sets. The table captions are below the tables in the paper.

Ah, I see. That makes a lot more sense

**Review by benj-zen — APPROVED** [review]

**Pull request merged**